content/doc/developer/internationalization/new-internationalization.adoc

```diff
@@ -0,0 +1,339 @@
+---
+title: New Internationalization
+layout: developersection
+---
+ 
+https://wiki.jenkins.io/display/JENKINS/How+to+view+Jenkins+in+your+language[How
+to view Jenkins in your language].
+
+'''''
+
+== Translations to Specific languages
+
+* https://wiki.jenkins.io/display/JENKINS/Chinese+%28Traditional%29+Translation[Chinese
+(Traditional) Translation]
+* https://wiki.jenkins.io/display/JENKINS/Chinese+%28Simplified%29+Translation[Chinese
+(Simplified) Translation] 
+* https://wiki.jenkins.io/display/JENKINS/French+Localization[French
+Localization]
+* https://wiki.jenkins.io/display/JA/Japanese+Localization[Japanese
+Localization]
+* https://wiki.jenkins.io/display/JENKINS/Polish+Localization[Polish
+Localization]
+* https://wiki.jenkins.io/display/JENKINS/Spanish+Translation[Spanish
+Translation]
+* https://wiki.jenkins.io/display/JENKINS/Brazilian+Portuguese+Translation[Brazilian
+Portuguese Translation]
+
+== What developers need to know/do
+
+Jenkins' i18n support has some aspects that developers need to be aware
+of.
+
+. Generation of type-safe `+Messages+` classes from
+`+Messages.properties.+`
+. Marking messages in jelly files.
+. Creation of html files for help tips.
+
+=== Generation of `+Messages+` classes
+
+Jenkins uses http://java.net/projects/localizer/[localizer] to generate
```

Comment:
```suggestion
Jenkins uses the Java https://docs.oracle.com/middleware/1212/wls/WLAPI/index.html?weblogic/i18n/Localizer.html/[lLocalizer] package to generate
```
The existing link is broken (java.net has been deprecated) so I chose a different page for this link -- please verify whether this is a good choice.

Useful to specify that this is a Java package rather than just show "localizer" in the text.  And this reference page capitalizes Localizer so I followed suit.